### PR TITLE
Use bigheap to inject heap stress

### LINKED ISSUE #737: memory in StressChaos is virtual memory but not real memory
## Bug Report

**What did you do?**

Apply StressChaos with memory stressors.

**What did you expect to see?**

Increase the memory usage and cause OOM.

**What did you see instead?**

Increase virtual memory usage, which is not a regular scenario.

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: Yang Keao <keao.yang@yahoo.com>

### What problem does this PR solve?

Fix #737 

### What is changed and how does it work?

Use `--bigheap` to inject memory stress rather than `--vm` which is virtual memory.

### Check List

Tests

- Manual test

Code changes

- Has Go code change

Side effects

- Breaking backward compatibility

Remove `Size` field of memory stressor, because we cannot set the total size of `bigheap` stressor, only the speed is configurable.

Maybe we should provide some options to choose the memory stressors between `brk`, `bigheap` and `stack`, do you think it is needed?

### CODE REVIEW

File: api/v1alpha1/stresschaos_types.go [hunk 1]
```diff
@@ -233,19 +232,7 @@ type Stressors struct {
...
-				if err != nil {
-					return "", err
-				}
-				stressors += fmt.Sprintf(" --vm-bytes %d", size)
-			} else {
-				stressors += fmt.Sprintf("--vm-bytes %s",
-					in.MemoryStressor.Size)
-			}
-		}
+		stressors += fmt.Sprintf(" --bigheap %d", in.MemoryStressor.Workers)
```
Comment (cwen0) [inline api/v1alpha1/stresschaos_types.go:235]: Can we control how much memory it takes up?

Comment (YangKeao) [reply api/v1alpha1/stresschaos_types.go:235]: No. The stress-ng doesn't provide such arguments. We can only control how fast it eats up ram (but these arguments are also hard to understand and calculate).

**Comment by cwen0** [comment]
@fewdan @g1eny0ung Do we also need to update Chaos Dashboard after this pr is merged?

**Comment by g1eny0ung** [comment]
@cwen0 Yep. Due to the memory `size` field will be unavailable. It's needed to change both the API and the dashboard interface.

**Comment by fewdan** [comment]
The dashboard needs to be modified, but not many changes.

**Pull request closed without merging**